blueoil/generate_lmnet_config.py

```diff
@@ -154,10 +162,10 @@ def _blueoil_to_lmnet(blueoil_config):
 
     learning_rate_func = None
     learning_rate_kwargs = None
-    if learning_rate_schedule == "constant":
+    if optimizer == "Momentum":
```

Comment:
It looks didn't care the case where "constant" and "Adam".

Reply:
When choose "constant" and "Adam", below are used.
```
             'LEARNING_RATE_FUNC': <function piecewise_constant at 0x7f0a11b89488>,
             'LEARNING_RATE_KWARGS': None,
             'OPTIMIZER_CLASS': <class 'tensorflow.python.training.adam.AdamOptimizer'>,
             'OPTIMIZER_KWARGS': {'learning_rate': 0.001},
```

`LEARNING_RATE_FUNC` should be set `None`

Reply:
@iizukak and I fixed this issue. Please review again.